api/app/fireweather_bot/hourly_actuals.py

```diff
@@ -118,6 +120,9 @@ def main():
     except Exception as exception:
         # Exit non 0 - failure.
         logger.error('Failed to retrieve hourly actuals.', exc_info=exception)
+        rc_message = ':scream: Encountered error retrieving hourly actuals\n{}: {}'.format(
```

Comment:
You'll need a unit test that asserts that send_rocketchat_notification is called when an exception is thrown and this section of code runs.